db/migrate/20160517184709_add_direct_otp_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddDirectOtpToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :direct_otp, :string
+    add_column :users, :direct_otp_sent_at, :datetime
+  end
+end
```

Comment:
Should we also add indexes on these columns since we'll be querying them often? Perhaps a unique index on `direct_otp`?